exampleSite/content/_index/portfolio-header.md

```diff
@@ -0,0 +1,8 @@
++++
+date = "2018-07-13"
+fragment = "section-header"
+weight = 740
+
+title = "Portfolio Fragment"
+subtitle = "Displaying animals with links and modals"
```

Comment:
We should probably rename that test data to make more sense and not just duplicate the portfolio data.